libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -827,6 +827,15 @@ float GCS_MAVLINK::telemetry_radio_rssi()
     return last_radio_status.rssi/254.0f;
 }
 
+uint8_t GCS_MAVLINK::get_last_txbuf()
```

Comment:
Do you mean the magic value 100?  That's not much magic.  Do you feel strongly about this?  I suppose we could do this, but it would complicate the logic a little where we use this function so I prefer it this way.

Reply:
> Do you feel strongly about this?

Yes.

Reply:
I replaced this code with a predicate function in my last commit so hopefully this resolves your concern.